apps/website-25/src/components/courses/exercises/FreeTextResponse.tsx

```diff
@@ -42,23 +61,23 @@ const FreeTextResponse: React.FC<FreeTextResponseProps> = ({
       </div>
       <div className="free-text-response__options flex flex-col gap-2">
         <textarea
+          {...register('answer')}
           className={`free-text-response__textarea p-4${
-            isSaved ? ' free-text-response__textarea--saved container-active bg-[#63C96533] border-[#63C965] text-[#2A5D2A]' : ' container-lined'
+            (!isEditing && exerciseResponse) ? ' free-text-response__textarea--saved container-active bg-[#63C96533] border-[#63C965] text-[#2A5D2A]' : ' container-lined'
           }`}
           placeholder="Enter your answer here"
-          onChange={handleAnswerChange}
+          onChange={() => setIsEditing(true)}
         />
       </div>
       <CTALinkOrButton
```

Comment:
consider: You might want to get `formState` from `useForm`. It has `formState.isSubmitting`, which you can use to disable the button and change the text to 'Saving' to give the user more feedback.